src/context/MapCenterContext.js

```diff
@@ -5,6 +5,7 @@ const MapCenter = ({ children }) => {
   const defaultCenterMap = {
     lat: 48.8566,
     lng: 2.3522,
+    isDragged: false,
```

Comment:
**Interesting:** Why did you add `isDragged` property to the existing state and not create a new, independent state?

Reply:
We introduced it to have a differentiator to say have these coords changed because we were dragging the map or have they changed because our location changed

Reply:
@katia-utochkina & @jssckbl: 

I understand that you need some state or 'global' variable to help you if a user drags the map or not. That's OK!

However, this specific `defaultCenterMap` variable: it's our default location (Paris - latitude, longitude), and it shouldn't change (in my opinion), it's a constant in our project. Another hand, our app allows us to change `newCenterMap` state, not the `defaultCenterMap`. 

In my humble opinion, it might cause some future problems (like `fly to me` button where `isDragged` is not declared or passed, and our map center is 'incomplete' because we 'forgot' to pass this property multiple times). I believe it has an important **'impact'** on our app. Also, with our current user story `search a place`: we should re-center the map but we don't matter if we dragged the map or not.

I suggest you can create a new, independent state `isDragged` because it's easier to use, change, and manage. But, if you want to ask it our mentors, you can do it.

Reply:
Echoing Alejandro, could @katia-utochkina or @jssckbl confirm that we don’t alter `defaultCenterMap` somewhere else in the code? Makes sense to me to have a sensible default value for `isDragged` as part of this object, but Alejandro is right that it should stay constant once it’s set the first time.

A way to guarantee that would be to use `Object.freeze` on it. That will surface very quickly whether a method is trying to alter it!

Reply:
@segdeha, I called an  ``Object.freeze()``  on it, console logged its props. ``defaultCenterMap`` stays the same during interactions with the map

Reply:
Great! Thanks for making sure!